### ISSUE TITLE
Add kubernetes stateful set for bootstrap nodes

### ISSUE THREAD

**Issue by geoah** [nimona/go-nimona#151]
* [x] Depends on #150 
* [x] Depends on #149 
* [ ] Add kubernetes stateful set resource for bootstrap nodes

Would be nice to have all peers be on the same port and have different hostnames, but not sure how we would go about this without elastic ips (assuming we go for digitalocean kube cluster).
Otherwise all nodes can share the same ip and listen of different ports, eg from 5000 to 5010.

@superdecimal any other ideas?
